data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/response/MembershipPersistenceResponse.avsc

```diff
@@ -18,14 +18,16 @@
         "net.corda.data.membership.db.response.command.PersistGroupParametersResponse",
         "net.corda.data.membership.db.response.command.PersistApprovalRuleResponse",
         "net.corda.data.membership.db.response.command.DeleteApprovalRuleResponse",
+        "net.corda.data.membership.db.response.command.RevokePreAuthTokenResponse",
```

Comment:
Should we have a response type for add token also, or it's not necessary?

Reply:
Not necessary. We just use a `MembershipPersistenceResult.success` on success and a `MembershipPersistenceResult.Failure` on failure.